buildsystem/cpp.cmake

```diff
@@ -5,6 +5,7 @@
 #TODO: integrate PGO (profile-guided optimization) build
 
 function(cpp_init)
+	set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
redundant with the root `CMakeLists.txt`, we probably want to keep it here though i guess.